packages/interfaces/src/store.ts

```diff
@@ -42,10 +44,19 @@ export type StoreQueryOptions = {
    * Cursor as an index to start a query from. Must be generated from a Waku
    * Message.
    */
-  cursor?: Cursor;
+  cursor?: proto.Index;
 };
 
-export interface IStore extends IBaseProtocolCore {
+export interface IStoreCore extends IBaseProtocolCore {}
```

Comment:
nit: as the other line uses `type` definition - makes sense to do it here as well

```suggestion
export type IStoreCore = IBaseProtocolCore;
```

Reply:
thanks! addressed: ed13dbf60c8ba85342a09b4532f166a17819681a